coral/src/app/router.tsx

```diff
@@ -10,11 +9,7 @@ const routes: Array<RouteObject> = [
     element: <HomePage />,
   },
   {
-    path: "/login",
-    element: <Login />,
-  },
-  {
-    path: "/topics",
+    path: "/browse-topics",
```

Comment:
😬  I renamed it because I was so annoyed with the current naming. 

The feature is called `topics`, which means it should export one component called `Topics`. 
But than the page component also was called `Topics` 🙈 

Maybe `topics-overview` would be a good feature name?